matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/model/RoomSummaryEntity.kt

```diff
@@ -40,12 +41,19 @@ internal open class RoomSummaryEntity(@PrimaryKey var roomId: String = "",
 ) : RealmObject() {
 
     private var membershipStr: String = Membership.NONE.name
+    private var versioningStateStr: String = VersioningState.NONE.name
+
 
     @delegate:Ignore
     var membership: Membership by Delegates.observable(Membership.valueOf(membershipStr)) { _, _, newValue ->
         membershipStr = newValue.name
     }
 
+    @delegate:Ignore
+    var versioningState: VersioningState by Delegates.observable(VersioningState.valueOf(versioningStateStr)) { _, _, newValue ->
```

Comment:
i think i remember we had an issue using delegates for two way binding?

Reply:
Why is there still membership so????